packages/modal/src/Modal/ModalView.tsx

```diff
@@ -87,6 +87,20 @@ const ModalView = React.forwardRef(
           fallbackFocus: `#${closeId}`, // tests fail without a fallback. (https://github.com/focus-trap/focus-trap-react/issues/91)
         };
 
+    let size: ModalSize = ModalSize.Default;
```

Comment:
Could you do something like, 
```
size = ModalSize[size] ?? ModalSize.Default
```

Reply:
nvm, i don't think thats correct

Reply:
or maybe something like this so you don't have to list every size

```
const exists = Object.values(ModalSize).includes(size);
const size = exists ? size : ModalSize.Default
```

Reply:
good call - i can do this